### ISSUE TITLE
No documentation on "Variable flags" or "Command flags"

### ISSUE THREAD

**Issue by Fluffkin** [sharkwouter/minigalaxy#464]
Looking in a game's properties gives these two fields.
I can't find any documentation on what they do.

**Comment by sharkwouter** [comment]
Good point, this should probably be improved.Variable flags get added in front of the command which is used to run the game, command flags after. A tooltip would be a good start.

**Comment by bclaesen** [comment]
Looking at the main branch right now, these tool tips seem to be present in `properties.ui` as you have described.

```
<object class="GtkEntry" id="entry_properties_variable">
    <property name="visible">True</property>
    <property name="can_focus">True</property>
    <property name="tooltip_text" translatable="yes">Added in front of the command used to launch the game</property>
    <property name="halign">start</property>
</object>
```
So not sure if you are looking for more updates here?

**Comment by Fluffkin** [comment]
It was more of a concern for wine programs. If "wine program" is the standard start, is it "<variable flags> wine program <command flags>", or "wine <variable flags> program <command flags>" ?
It would actually be useful to be able to specify "<variable flags> wine <wine commands> program <command flags>", assuming that's how it works. Giving access to /desktop...

**Comment by bclaesen** [comment]
Do you have any examples of `<wine commands>`? All I can find is the following:

```
Usage: wine PROGRAM [ARGUMENTS...]   Run the specified program
       wine --help                   Display this help and exit
       wine --version                Output version information and exit
```
which indicates there might not be any, but you seem to hint towards passing arguments?

Otherwise the syntax would indeed be
`<variable flags> wine program <command flags>`

That could be documented with examples indeed (such as using `WINEDLLOVERRIDES` in variable flags, and `-screen-width 3840` for command flags.

**Comment by Fluffkin** [comment]
The main gaming use is [`wine explorer /desktop=name,1024x768 program.exe`](https://wiki.winehq.org/FAQ#How_do_I_get_Wine_to_launch_an_application_in_a_virtual_desktop.3F) as an example.

The name is required otherwise all wine virtual desktops have the same naming. If you're using something like KDE / Devilspie2 / Openbox to manipulate window size / position / decorations based on name, it's vital to allow distinguishing between applications.

Unfortunately, some wine games do require running in a virtual desktop to obtain a reasonable window / resolution.

**Comment by bclaesen** [comment]
Ah, got it! Thank you.

In my understanding that might not be possible with the current way these variables are used:

https://github.com/sharkwouter/minigalaxy/blob/729d3439a585ab50378bc5f35bedda5f3d1886ba/minigalaxy/launcher.py#L96
where `exe_cmd` is either `wine <X>` or `wine start /b /wait /d <XworkingDir> <X>` for wine games.

Documentation could indicate that `winecfg` should be used to set a virtual desktop, but as setting the name per instance, that seems like a new feature (though I am new to the source code :)).
